src/libs/OptionsListUtils.ts

```diff
@@ -1025,22 +1076,21 @@ function getTagListSections(tags, recentlyUsedTags, selectedOptions, searchInput
         return tagSections;
     }
 
-    const selectedOptionNames = _.map(selectedOptions, (selectedOption) => selectedOption.name);
-    const filteredRecentlyUsedTags = _.map(
-        _.filter(recentlyUsedTags, (recentlyUsedTag) => {
-            const tagObject = _.find(tags, (tag) => tag.name === recentlyUsedTag);
-            return Boolean(tagObject && tagObject.enabled) && !_.includes(selectedOptionNames, recentlyUsedTag);
-        }),
-        (tag) => ({name: tag, enabled: true}),
-    );
-    const filteredTags = _.filter(enabledTags, (tag) => !_.includes(selectedOptionNames, tag.name));
-
-    if (!_.isEmpty(selectedOptions)) {
-        const selectedTagOptions = _.map(selectedOptions, (option) => {
-            const tagObject = _.find(tags, (tag) => tag.name === option.name);
+    const selectedOptionNames = selectedOptions.map((selectedOption) => selectedOption.name);
+    const filteredRecentlyUsedTags = recentlyUsedTags
+        .filter((recentlyUsedTag) => {
+            const tagObject = tags.find((tag) => tag.name === recentlyUsedTag);
+            return !!tagObject?.enabled && !selectedOptionNames.includes(recentlyUsedTag);
+        })
+        .map((tag) => ({name: tag, enabled: true}));
+    const filteredTags = enabledTags.filter((tag) => !selectedOptionNames.includes(tag.name));
+
+    if (selectedOptions) {
```

Comment:
This caused regression. `selectedOptions` is array and always truthy.
Should be `selectedOptions.length`.